sdk/webpubsub/Azure.Messaging.WebPubSub/src/Generated/WebPubSubServiceClient.cs

```diff
@@ -1999,7 +1999,7 @@ internal HttpMessage CreateSendToUserRequest(string userId, RequestContent conte
             return message;
         }
 
-        internal HttpMessage CreateAddUserToGroupRequest(string group, string userId, RequestContext context)
+        internal HttpMessage CreateAddUserToGroupRequest(string userId, string group, RequestContext context)
```

Comment:
Parameters are reordered because of how they are ordered in the path
```cs
            uri.AppendPath("/users/", false);
            uri.AppendPath(userId, true);
            uri.AppendPath("/groups/", false);
            uri.AppendPath(group, true);
```

Reply:
We'll need to make sure this change is well-publicized in the next beta release.  Would you also update the changelog for this library in this PR?

Reply:
Oh nevermind, this is an internal method -- I see your note in Extensions